### PR TITLE
Men Without Women stories added for public domain day

### PR DESCRIPTION
**Pull request by maticstric**

### CODE REVIEW

File: src/epub/css/core.css [hunk 1]
```diff
@@ -99,12 +99,6 @@ i > em{
 	font-style: normal;
 }
 
-ol,
```
Comment (acabal) [inline src/epub/css/core.css]: Please add these back in as they are part of the new template. Thanks

File: src/epub/css/local.css [hunk 2]
```diff
@@ -1,6 +1,12 @@
 @charset "utf-8";
 @namespace epub "http://www.idpf.org/2007/ops";
 
+header{
+	font-variant: small-caps;
+	margin: 1em;
+	text-align: center;
```
Comment (acabal) [inline src/epub/css/local.css]: text-align: center is not necessary because header gets that from core.css

**Review by acabal — CHANGES_REQUESTED** [review]
Looks great! Just a few minor notes in the CSS.

**Comment by maticstric** [comment]
Should be done!

**Comment by acabal** [comment]
Excellent, thanks!

**Pull request merged**